jmclient/jmclient/configure.py

```diff
@@ -419,14 +422,28 @@ def remove_unwanted_default_settings(config):
         if section.startswith('MESSAGING:'):
             config.remove_section(section)
 
-
-def load_program_config(config_path=None, bs=None):
+def load_program_config(config_path="", bs=None):
     global_singleton.config.readfp(io.StringIO(defaultconfig))
-    remove_unwanted_default_settings(global_singleton.config)
     if not config_path:
-        config_path = os.getcwd()
+        config_path = lookup_appdata_folder(global_singleton.APPNAME)
+    # we set the global home directory, but keep the config_path variable
+    # for callers of this function:
+    global_singleton.datadir = config_path
+    jmprint("User data will be stored and accessed in this location: " + \
```

Comment:
This message looks ok on a first run when datadir is created, but afterwards that future tense ("will") does not sound right to me. Maybe just "User data location: "?

Reply:
Yes good point. I was unsure about this message but change from "will be stored" to "stored" seems logical.
Perhaps this message should be removed at some point, but I like the idea of having it for now, because people will be transitioning and they will find a reminder helpful.
Agreed?

Reply:
Probably in some future this message could be removed unless `--datadir` is specified as an argument to the script.